docs/how-to/account.md

```diff
@@ -0,0 +1,8 @@
+# How to create an OpenCTI user account
+
+The value of the charm configuration `admin-user` contains the login information
+for the admin account. Other accounts, including separate admin accounts, can 
+be created using the initial admin account by following the OpenCTI platform
+tutorial.
+
+https://docs.opencti.io/latest/administration/users/#manage-a-user
```

Comment:
Possible missing newline at the end of file

Reply:
Added, thanks!